scipy/stats/_multivariate.py

```diff
@@ -1356,8 +1356,9 @@ class dirichlet_gen(multi_rv_generic):
     support on the simplex defined by
 
     .. math::
-        \sum_{i=1}^{K} x_i \le 1
+        \sum_{i=1}^{K} x_i = 1 where x_i \in (0,1)
```

Comment:
Check the link to the CircleCI "artifact" to see how this will look when rendered.  You'll see that it doesn't render "where" correctly.  This can be handled with more LaTeX markup, but it will be easier to read the text in terminal if the "where" clause is continued outside of the `math` environment.  Perhaps something like this:

```suggestion
        \sum_{i=1}^{K} x_i = 1

    where 0 < x_i < 1.
```

Reply:
Oh, I’m sorry I didn’t realized that. I accepted your suggestion.